### DISCUSSION TITLE
a-baseline-shift-002.svg

### DISCUSSION THREAD

**Discussion by paulushub** [linebender/resvg-test-suite#32] [General]
**Test File**:
[a-baseline-shift-002.svg](https://github.com/RazrFalcon/resvg-test-suite/blob/master/svg/a-baseline-shift-002.svg)
```xml
<svg id="svg1" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"
     font-family="Noto Sans" font-size="26">
    <title>`sub`</title>

    <path id="crosshair" d="M 20 100 L 180 100 M 100 20 L 100 180"
          stroke="gray" stroke-width="0.5"/>

    <text id="text1" x="28" y="100">
        Text<tspan id="tspan1" fill="green" baseline-shift="sub">Text</tspan>Text
    </text>

    <!-- image frame -->
    <rect id="frame" x="1" y="1" width="198" height="198" fill="none" stroke="black"/>
</svg>
```
**Expected Output**:
[a-baseline-shift-002.png](https://github.com/RazrFalcon/resvg-test-suite/blob/master/png/a-baseline-shift-002.png)

**Issue**:
The output does not appear to show a **subscript attribute**: `baseline-shift="sub"`
Batik 1.14 and Chrome-based outputs, however, agree with my expected result.

**Comment by RazrFalcon** [comment]
Yes, the baseline-shift value is unclear. Some implementations use a font-based value, some fixed offset. I'm using font one and looks like the font I've choose has a very small one.

Would have to double check it. Maybe this is some resvg bug.

**Comment by RazrFalcon** [comment]
I thought maybe Noto Sans uses the [BASE](https://docs.microsoft.com/en-us/typography/opentype/spec/base) table, which resvg doesn't support, but it's not.
